hmpl-lang/hmpl-lang.github.io
# Add Scroll To Top Button with Custom SVG

## Description

This pull request enables the `backToTop` plugin in VuePress and using a custom SVG instead of the default one.

## Changes Made

1. Installed the `@vuepress/plugin-back-to-top` plugin.
2. Updated `docs/.vuepress/config.js` to include the `backToTop` plugin.
3. Added a custom SVG file in images.
4. Added custom CSS to style the back-to-top button.

## How to Test

1. Run `npm install` to ensure the new plugin is installed.
2. Build the VuePress site using `npm run build`, followed by the start command `npm run dev`.
3. Scroll down on any page to see the custom back-to-top button.
4. Click the button to ensure it scrolls the page back to the top.

## Related Issue

Fixes #9 

## Screenshots

![image](https://github.com/user-attachments/assets/5bdb139d-7bb7-4d39-bd52-291490d4311e)
